features/polyExplorer/src/components/dataViz/purposesBarChart.jsx

```diff
@@ -145,18 +147,18 @@ const PurposesBarChart = ({ data, animation, onClick }) => {
                 .enter()
                 .append("foreignObject")
                 .attr("x", labelXPosition)
-                .attr("y", (d) => yScale(d.title) + gHeight)
+                .attr("y", (d) => yScale(d.title) + labelTitleYAdjust)
                 .attr("width", width)
-                .attr("height", labelTitleHeight.fo)
+                .attr("height", 36)
                 .on("click", (_, d) => onClick(d));
             const labelTitleDiv = labelTitle.append("xhtml:div");
             const labelTitleP = labelTitleDiv
                 .append("p")
                 .text((d) => d.translation || d.title)
-                .attr("height", labelTitleHeight.p)
                 .style("margin", legendMargin)
                 .style("display", "inline-block")
-                .style("font-size", fontConfig.fontSizeTitle)
+                .style("box-sizing", "border-box")
+                .style("height", labelTitleHeight.p + "px")
```

Comment:
Naming nit: could `p` be renamed to `pixels`?

Reply:
"p" is as `paragraph`, could be rename as `paragraph` if it makes it clearer

Reply:
`paragraphHeight`, maybe? Or, since it's an attribute for `labelTitleHeight`, `forParagraphs`